### PR TITLE
Fixing TestOKenabled and reset of ucrReceiver in the dlg Enter

### PR DESCRIPTION
**Pull request by stevenndungu**

### CODE REVIEW

**Comment by lilyclements** [comment]
This looks ready for Danny to review

File: instat/dlgEnter.vb [hunk 1]
```diff
@@ -56,13 +56,18 @@ Public Class dlgEnter
         ucrDataFrameEnter.Reset()
         chkSaveEnterResultInto.Checked = True
         ucrSaveEnterResultInto.SetPrefix("Enter")
+        ucrReceiverForEnterCalculation.Clear()
```
Comment (dannyparsons) [inline instat/dlgEnter.vb]: Clearing the receiver should be done automatically when you reset the selector. You shouldn't have to clear a receiver directly.

File: instat/dlgEnter.vb [hunk 2]
```diff
@@ -263,7 +268,7 @@ Public Class dlgEnter
         TestOKEnabled()
     End Sub
 
-    Private Sub ucrSaveEnterResultInto_NameChanged() Handles ucrSaveEnterResultInto.NameChanged
+    Private Sub ucrSaveEnterResultInto_ContentsChanged() Handles ucrSaveEnterResultInto.ContentsChanged
```
Comment (dannyparsons) [inline instat/dlgEnter.vb]: You shouldn't be doing SaveResults() on ContentsChanged, that should be on NameChanged.
Doing TestOK on ContentsChanged is sensible. You can have two different subs if needed.

**Comment by lilyclements** [comment]
Fixes #2080 
Fixes #1751

File: instat/dlgEnter.vb (hunk 1 above)
Comment (stevenndungu) [inline instat/dlgEnter.vb]: But I thought that this Data frame selector is slightly different. In the sense that the receiver is not connected to the data frame selector. The receiver gets information from the  buttons of functions and characters below it. 

From the testing I have made, even after resetting the data frame  the contents of the receiver are not cleared.

**Pull request merged**